doc/source/examples/03_modeling/design_parameters.mystnb

```diff
@@ -0,0 +1,145 @@
+---
+jupytext:
+  text_representation:
+    extension: .mystnb
+    format_name: myst
+    format_version: 0.13
+    jupytext_version: 1.16.4
+kernelspec:
+  display_name: Python 3 (ipykernel)
+  language: python
+  name: python3
+---
+
+# Modeling: Using Design Parameters
+
+You can read and update parameters that are part of the design.
+In this example, we have a simple design with two associated parameters.
```

Comment:
```suggestion
In this example, the simple design has two associated parameters.
```
Avoid "we" and use active voice.

Reply:
Thanks for the reviews @PipKat, I used "we" based on the previous examples on this examples section, so maybe they need to be corrected as well? thanks
 https://github.com/ansys/pyansys-geometry/blob/150c19b9dabe92e999262f89233970efc5c6c96a/doc/source/examples/01_getting_started/05_plotter_picker.mystnb#L177C23-L177C30

Reply:
Here is the guideline: Avoid first-person pronouns (I, we, us, our, and ours), For more information, see https://developers.google.com/style/pronouns#personal-pronouns.

Reply:
Hi @umutsoysalansys - correct, @PipKat tries to review our examples on a best effort basis, but she might have missed the one you linked. I will review that one myself so that it is in-line with the style guidelines. Thank you as always @PipKat!